yarn-project/cli/package.json

```diff
@@ -72,6 +72,7 @@
     "@aztec/constants": "workspace:^",
     "@aztec/foundation": "workspace:^",
     "@aztec/l1-artifacts": "workspace:^",
+    "@aztec/p2p": "workspace:^",
```

Comment:
should items be re-exported from p2p bootstrap rather than exposing p2p into the cli